src/ConnectionManager.js

```diff
@@ -34,8 +34,9 @@ define(["structures/Response", "structures/Request", "structures/CAPIError"],
      * @param [body=""] {String} a string which should be passed in request body to the REST service
      * @param [headers={}] {object} object literal describing request headers
      * @param callback {Function} function, which will be executed on request success
+     * @param xhrCallbacks {Object} a set of callbacks to apply on a specific XHR event like onload, onerror, onprogress, etc.
      */
-    ConnectionManager.prototype.request = function (method, url, body, headers, callback) {
+    ConnectionManager.prototype.request = function (method, url, body, headers, callback, xhrCallbacks) {
```

Comment:
I think you can still avoid BC break even if you put the new parameter before `callback` with one more else if to detect the callback parameter